src/styles/index.scss

```diff
@@ -1,3 +1,149 @@
-body {
+$main-accent: #060b35;
+$secondary: #616070;
+$blue-accent: #00acdc;
+$white: white;
+$starnum: 1,2,3,4;
+
+html,
+body,
+div,
+span,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p {
```

Comment:
do you need to normalize all of them? 
leave only those which you use

also move it to the other file

Reply:
done, see 'v2'